pkg/util/client.go

```diff
@@ -0,0 +1,28 @@
+package util
```

Comment:
+Copyright

Reply:
let me remove this file. It's not used now.